hkm/templates/hkm/base_viewer.html

```diff
@@ -35,32 +35,34 @@
         </button>
         {% if user.is_authenticated %}
           {% if collection_record %}
-          <button class="actions__btn actions__btn--right nav__fav {% if is_favorite %}active{% endif %}" data-record-id="{{ hkm_id }}" id="popover-favorite">
+            <button class="actions__btn actions__btn--right nav__fav {% if is_favorite %}active{% endif %}" data-toggle="tooltip" data-placement="top" tooltip-title="{% if is_favorite %}{% trans 'Remove from favorites' %}{% else %}{% trans 'Add to favorites' %}{% endif %}" data-record-id="{{ hkm_id }}" id="popover-favorite" >
```

Comment:
I wondered if these buttons would be accessible after the tooltip is added and it appears so!